MANIFEST.in

```diff
@@ -2,6 +2,10 @@ include README.md
 include LICENSE
 
 # Included in PyPI distribution
+include bentoml/*.py
```

Comment:
@aarnphm these lines here are not needed

Reply:
all files tracked by git is already included by default,  now with setuptools scm

Reply:
noted